Add classic project (#240)
plone/cookieplone-templates: templates/sub/classic_project_settings/README.md

# Cookieplone project settings

[![Cookieplone Templates: CI](https://github.com/plone/cookieplone-templates/actions/workflows/main.yml/badge.svg)](https://github.com/plone/cookieplone-templates/blob/main/.github/workflows/main.yml)
[![Built with Cookieplone](https://img.shields.io/badge/built%20with-Cookiecutter-ff69b4.svg?logo=cookiecutter)](https://github.com/plone/cookieplone-templates/)
[![License](https://img.shields.io/github/license/plone/cookieplone-templates)](../../../LICENSE)
[![Black code style](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)

> This is a sub-template—used by other templates, in automated tests, and for OCI image generation—accessible with **Cookieplone** at the path `sub/classic_project_settings`.

Reviewer: ```suggestion
This is a subtemplate—used by other templates, in automated tests, and for OCI image generation—accessible with **Cookieplone** at the path `sub/classic_project_settings`.
```

### uv

To install uv, use the following command, or visit the [uv installation page](https://docs.astral.sh/uv/getting-started/installation/) for alternative methods.

```shell
curl -LsSf https://astral.sh/uv/install.sh | sh
```

## Generate your project settings 🎉

```shell
uvx cookieplone sub/project_settings
```

### Project generation options 🛠️

The table below describes the options you can customize using the [Cookiecutter CLI](https://github.com/cookiecutter/cookiecutter) during the generation process.

| Option                | Description                                                                                                                                          | Example                       |
| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- |
| `title`  | Your project's human-readable name, capitals and spaces allowed.                                                                                     | `Project Settings`                |

> | `author`              | This is you! The value goes into places like `LICENSE`, `package.json` and such.                                                                     | `Our Company`               |

Reviewer: ```suggestion
| `author`              | This is you! Its value goes into `LICENSE`, `setup.py`, and other files.                                                                     | `Our Company`               |
```

> | `email`               | The email address you want to identify yourself in the project.                                                                                      | `email@example.com`         |

Reviewer: ```suggestion
| `email`               | The email address to contact the project maintainer.                                                                                      | `email@example.com`         |
```
